lib/mutant/ast/regexp/transformer/alternative.rb

```diff
@@ -0,0 +1,27 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        class Alternative < self
+          ALTERNATIVE = IceNine.deep_freeze(::Regexp::Expression::Alternative.new(0, 0, 0))
+
+          register :regexp_sequence_expression
+
+          class ExpressionToAST < Transformer::ExpressionToAST
+            def call
+              ast(*children)
+            end
+          end
+
+          class ASTToExpression < Transformer::ASTToExpression
+            def call
+              root = ALTERNATIVE.dup
```

Comment:
Since this is always acting on `root` and returning the object from this method what about using `Object#tap`? There's a few places in this change where this could be used.

Although I am not sure how @mbj feels about that in mutant code; I don't know if he uses it elsewhere.